usaspending_api/database_scripts/etl/award_delta_view.sql

```diff
@@ -92,6 +95,79 @@ LEFT JOIN (
   FROM
     recipient_lookup AS rlv
 ) recipient_lookup ON (recipient_lookup.duns = vw_award_search.recipient_unique_id AND vw_award_search.recipient_unique_id IS NOT NULL)
+LEFT JOIN (
+    WITH closed_periods AS (
+	    -- This is NOT right, since period dates are not the same as submission window close dates,
+	    -- but an approximation for illustration purposes until we have the reporting dates table up
+	    SELECT DISTINCT concat(p.reporting_fiscal_year::text,
+	        lpad(p.reporting_fiscal_period::text, 2, '0')) AS fyp,
+	        p.reporting_fiscal_year, p.reporting_fiscal_period
+	    FROM submission_attributes p
+	    WHERE
+	    -- For COVID, look only after a certain date (likely on/after 2020-04-01, or the start of FY2020 FYP07)
+	        p.reporting_period_start >= '2017-04-01' -- '2020-04-01' -- using older date to see data surface
+	        AND now()::date > p.reporting_period_end -- change "period end" with "window close"
+	    ORDER BY p.reporting_fiscal_year DESC, p.reporting_fiscal_period DESC
+    ),
+    eligible_submissions AS (
+        SELECT *
+        FROM submission_attributes s
+        WHERE concat(s.reporting_fiscal_year::text, lpad(s.reporting_fiscal_period::text, 2, '0')) IN (
+            SELECT fyp FROM closed_periods
+        )
+    ),
+    eligible_file_c_records AS (
+        SELECT
+            faba.award_id,
+            CASE WHEN faba.disaster_emergency_fund_code IN ('L','M','N','O','P') THEN faba.gross_outlay_amount_by_award_cpe else 0 END AS gross_outlay_amount_by_award_cpe,
+            CASE WHEN faba.disaster_emergency_fund_code IN ('L','M','N','O','P') THEN faba.transaction_obligated_amount else 0 END AS transaction_obligated_amount,
```

Comment:
There is a strong desire to prohibit hardcoding the COVID-19 DEFC into code. the lookup table provides a `covid_19` group which can be used to obtain the list.